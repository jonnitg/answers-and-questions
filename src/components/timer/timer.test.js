import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Timer from './timer';

it('renders timer correctly', () => {
  const { getByText } = render(<Timer seconds={60} />);
  const spanElement = getByText(/min/i);
  expect(spanElement).toHaveTextContent('1min. 0segs.');
});

it('matches snapshot', () => {
  const tree = renderer.create(<Timer seconds={60} />).toJSON();
  expect(tree).toMatchSnapshot();
});
