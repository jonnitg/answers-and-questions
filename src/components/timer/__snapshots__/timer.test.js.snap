// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<span
  className="sc-AxjAm khGDyj"
>
  1min. 0segs.
</span>
`;
